### ISSUE TITLE
Add method to set sentinel values

### ISSUE THREAD

**Issue by esheldon** [LSSTDESC/healsparse#137]
Ideas:

```
1. `hmap.update_values_pos/pix` with value set to None sets sentinal
2. make the sentinal public so we can send value to the above
3. `hmap.set_to_sentinal(pixels or indices?)`  # new method
```

**Comment by erykoff** [comment]
The reason `sentinel` isn't public is because of the way it interacts with recarray type maps.  Though I think I should probably make the value public anyway.

As a matter of convenience, I think that option 1 makes sense to me as a way of saying "unset these pixels".

**Comment by esheldon** [comment]
fine with me

**Comment by erykoff** [comment]
Fixed in #138
